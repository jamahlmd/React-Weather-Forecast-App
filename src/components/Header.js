import React from 'react';
import { Row, Col } from 'antd';
import propTypes from 'prop-types';



const Header = ({title}) => (

    <div>
        <Row style={styles.headerStyles}>
            <Col md={{span: 6, offset:6}} xs={{span: 20, offset:2}}><h1>{title}</h1></Col>
        </Row>
    </div>

);

const styles = {
    headerStyles: {
        backgroundColor: '#ADD6FF',
        paddingTop: 10,
    }
};

Header.propTypes = {
    title: propTypes.string.isRequired
};

export default Header;