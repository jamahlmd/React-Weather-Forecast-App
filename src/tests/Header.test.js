import React from 'react';
import Header from '../components/Header';

const wrapper = shallow(<Header title={"Test title"}/>);


test('Should render Header correctly', () => {
    expect(toJson(wrapper)).toMatchSnapshot();
});


test('Should render title correctly', () => {
    expect(wrapper.find('h1').text()).toBe('Test title');
});