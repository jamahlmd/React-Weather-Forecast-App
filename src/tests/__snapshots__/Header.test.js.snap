// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Header correctly 1`] = `
<div>
  <Row
    gutter={0}
    style={
      Object {
        "backgroundColor": "#ADD6FF",
        "paddingTop": 10,
      }
    }
  >
    <Col
      md={
        Object {
          "offset": 6,
          "span": 6,
        }
      }
      xs={
        Object {
          "offset": 2,
          "span": 20,
        }
      }
    >
      <h1>
        Test title
      </h1>
    </Col>
  </Row>
</div>
`;
